package.json

```diff
@@ -42,12 +42,12 @@
     "@guardian/src-brand": "^2.4.0",
     "@guardian/src-button": "^2.4.0",
     "@guardian/src-ed-lines": "^2.4.0",
-    "@guardian/src-foundations": "^2.4.0",
+    "@guardian/src-foundations": "^1.5.0",
     "@guardian/src-icons": "^2.4.0",
-    "@guardian/src-radio": "^2.4.0",
+    "@guardian/src-radio": "^1.5.0",
```

Comment:
Would it be better to fix this upstream in `atoms-rendering` rather than downgrade our version of Source?

Reply:
That could be possible. I also had to downgrade src-foundations to 2.0.0 on the discussion-rendering branch to be compatible, so there might be more work involved in updating a few shared libraries